src/components/teampage/team-member.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import {
+  ContactCardContainer,
+  CardPosition,
+  // FlexContainer,
+  MemberImg,
+  TextOverlay,
+  PositionContainer,
+  EmailText,
+  Text,
+  Email,
+  Overlay,
+  InfoContainer,
+  MemberName,
+  MemberPosition,
+  MemberMajor,
+  ConnectButton,
+  MemberInfoSpacer,
+  ConnectLink,
+} from "../teampage/team-member-styles"
+
+export default ({
+  name,
+  photo,
+  email,
+  major,
+  position,
+  linkedinUrl,
+  team,
+  index,
+  isLast,
+}) => (
+  <CardPosition index={index} isLast={isLast}>
+    <ContactCardContainer index={index}>
+      <MemberImg photoUrl={photo.fluid.src}></MemberImg>
+      {/* <Overlay>
+        <TextOverlay>
+          <PositionContainer>{position}</PositionContainer>
+          {major}
+          <EmailText>
+            <Text>Reach me at </Text>
+            <Email href={`mailto:${email}`} target="_blank">
+              {email}
+            </Email>
+          </EmailText>
+        </TextOverlay>
+      </Overlay> */}
+      <InfoContainer>
+        <MemberInfoSpacer></MemberInfoSpacer>
+        <div>
+          <MemberName>{name}</MemberName>
+          <MemberPosition>{position}</MemberPosition>
+          <MemberMajor></MemberMajor>
+        </div>
+        {linkedinUrl ? (
+          <ConnectLink href={linkedinUrl}>
```

Comment:
Instead of rendering an empty element, can you have this styled component take an extra prop to condition how its styles work? That way you save an element